kube/elasticsearch/elasticsearch-pv-snapshot.yaml

```
@@ -0,0 +1,21 @@
+
+---
+
+# VolumeSnapshot
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: es-pv-snapshot
+spec:
+  volumeSnapshotClassName: do-snapshot-class
+  source:
+    persistentVolumeClaimName: es-pvc
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)